.gitignore

```diff
@@ -15,3 +15,11 @@ src/Visualization/Shader/Shader.h
 _build/
 doxygen/
 .vs/
+src/open3d_config.h
+util/pip_package/setup.py
```

Comment:
Why do we need to add these two files in .gitignore? I understand src/open3d_config.h is generated from source, but setup.py is not.

Reply:
actually _setup.py_ is generated as well from _setup.py.in_.
I just double checked and _setup.py_ is not committed to the repository.

Reply:
Aha got it.